lib/paperclip/thumbnail.rb

```diff
@@ -118,4 +119,4 @@ def identified_as_animated?
       raise Paperclip::Errors::CommandNotFoundError.new("Could not run the `identify` command. Please install ImageMagick.")
     end
   end
-end
+end
```

Comment:
I'll fix this in an update, i removed the white space in the wrong part of the file